calculator.rb

```diff
@@ -0,0 +1,75 @@
+
+data = [
+  4,
+  "6",
+  { result: "Deset"}
+]
+
+
+puts "#{data[0]} + #{data[1]} = #{data[2][:result]} "
+
+
+
+
+class Calculator 
```

Comment:
please write spec test for Calculator class inside lib/classes/calculator_spec.rb and test following scenarios

https://www.youtube.com/watch?v=LdutLs5-ObI
https://www.youtube.com/watch?v=71eKcNxwxVY